python/ray/__init__.py

```diff
@@ -259,10 +258,12 @@ def __getattr__(self, attr):
     "PlacementGroupID",
 ]
 
+
 if sys.version_info < (3, 7):
     # TODO(Clark): Remove this one we drop Python 3.6 support.
     from ray import data  # noqa: F401
     from ray import workflow  # noqa: F401
+    from ray import autoscaler  # noqa: E402,F401
```

Comment:
What's the point in this? the autoscaler doesn't have any expensive imports does it? (my understanding is that torch, tf, arrow, etc are the expensive imports)

Reply:
autoscaler imports cli_logger which performs the monkeypatch. This will be broken after `ray.autoscaler.request_resources(num_cpus=1)` called. But 99% users won't use it.

Reply:
Btw I am considering multiple options. I believe it is the easiest solution, but we can do other things like explicitly require cli_logger to be obtained via function call (get_cli_logger()). or do monkey patch when the API is first called from cli logger.

Reply:
Anyway I think it is better doing lazy import since it is not necessary for any of ray core work, and I don't know what kind of other random monkey patching is happening under the hood.